### DISCUSSION TITLE
UNXZ installazione hassos su Mini PC

### DISCUSSION THREAD

**Discussion by mariellamarco** [indomus/forum#2624] [HUB: Home Assistant]
Ciao a tutti, 
sto installando home assistant su un minipc seguendo la guida. Quando provo a decomprimere l'immagine di Home Assistant OS con il comando `unxz hassos.iso.xz` mi esce l'errore `Errore di scrittura: Spazio esaurito sul device`. 

Qualcuno è in grado di risolvere questo problema? Se si come? 
Grazie
Marco

**Comment by KC-inDomus** [comment]
> Ciao a tutti, sto installando home assistant su un minipc seguendo la guida. Quando provo a decomprimere l'immagine di Home Assistant OS con il comando `unxz hassos.iso.xz` mi esce l'errore `Errore di scrittura: Spazio esaurito sul device`.
> 
> Qualcuno è in grado di risolvere questo problema? Se si come? Grazie Marco

hai scelto una chiavetta USB troppo piccola. è indicato chiaramente in guida

**Reply by mariellamarco** [reply]
La chiavetta che utilizzo è da 32gb creata con Balena Etcher.

**Reply by mariellamarco** [reply]
Ieri avevo lo stesso problema con una chiavetta da 8, ho pensato anche io fosse un problema di chiavetta e ne sto usando una da 32. È per quello che non riesco a capire quale possa essere il problema :)

**Reply by KC-inDomus** [reply]
allora prima del comando curl fai

cd /var/crash

e poi esegui curl e unxz

**Comment by mariellamarco** [comment]
Grazie mille @KC-inDomus funziona! 🚀

**Comment by Starc73** [comment]
mi collego a questo post perchè ho lo stesso problema, riepilogo i passaggi:
1) installata ubuntu 18.04 su chiavetta da 16 gb con balenaetcher
2) faccio partire la live e con terminal digito "sudo apt update -y && sudo apt install curl -y"
3) finito l'installazione precedente digito "curl -L 'https://indomus.it/link/home-assistant-os-x86-64' --output /var/crash/hassos.iso.xz"
4) finito lo scarico del file di c.a. 406MB digito "cd /var/crash" e poi "unxz hassos.iso.xz"
e mi appare il messaggio  spazio errore spazio esaurito sul device"
Seguendo quindi il consiglio di KC-inDomus relativo al penultimo post
5) essendo già in /var/crash digito "curl -L 'https://indomus.it/link/home-assistant-os-x86-64' --output /var/crash/hassos.iso.xz"
6) dopo lo scarico digito "unxz hassos.iso.xz"

ma niente sempre lo stesso errore spazio esaurito sul device. Eppure la chiave ha ancora tanto spazio libero

Chi mi può consigliare ? Grazie

**Reply by KC-inDomus** [reply]
da quanto è la chiavetta?

**Reply by Starc73** [reply]
16 GB

**Reply by indomus** [reply]
ribatti [la guida](https://indomus.it/guide/installare-e-configurare-home-assistant-os-su-un-mini-pc-intel-nuc-dedicato/) e usa la nuova immagine ubuntu che abbiamo linkato. abbiamo provato ieri notte, funziona tutto (su chiavetta da 16)

**Comment by Starc73** [comment]
Perfetto ho provato ieri sera e funziona, grazie 1000

Il giorno mar 23 lug 2024 alle ore 10:11 inDomus ***@***.***>
ha scritto:

**Reply by snakuzzo** [reply]
Per cortesia, non rispondere più alle discussioni facendo reply direttamente dalla mail.
Grazie.
